test/unit/lib/classes/Variables.test.js

```diff
@@ -2669,78 +2669,6 @@ module.exports = {
         .should.become('${deep:0.veryDeep}');
     });
   });
-  describe('#warnIfNotFound()', () => {
```

Comment:
Let's keep those tests

Reply:
How do you mean? we've removing `#warnIfNotFound`, so we can't keep these tests?

Reply:
In my understanding we followed this: https://github.com/serverless/serverless/pull/8710#discussion_r552058743. So we've renamed it and introduced a second mode in it internally.

Ideally those tests may be untouched and should remain passing.

Reply:
So, again, I believe those tests should be kept. As we didn't replace them with ones that cover same scenarios, and we didn't remove the functionality their testing